internal/template/merge_request_stacked_list.templ

```diff
@@ -0,0 +1,145 @@
+package template
+
+import (
+  "github.com/dxta-dev/app/internal/data"
+  "github.com/dxta-dev/app/internal/util"
+
+  "fmt"
+  "time"
+)
+
+type MergeRequestStackedListProps struct {
+  MergeRequests []data.MergeRequestListItemData
+  TimeNow time.Time
+  ShowTime bool
+}
+
+templ mergeRequestStack(title string, time string, timeAgo string, mrUrl string, canonId int64, avatarUrls []string, showTime bool, codeAdditions int64, codeDeletions int64) {
+	<li class="flex flex-wrap items-center justify-between gap-x-6 gap-y-4 py-5 sm:flex-nowrap">
+		<div>
+			<p class="text-sm font-semibold leading-6 text-gray-900">
+				<a href={ templ.SafeURL(mrUrl) } class="hover:underline">{ title }</a>
+			</p>
+      
+			<div class="mt-1 flex items-center gap-x-2 text-xs leading-5 text-gray-500">
+				<p>
+					<a href="#" class="hover:underline">#{fmt.Sprintf("%v", canonId)}</a>
+				</p>
+        <p>
+          <span class="text-lime-700 select-none">+{fmt.Sprintf("%v", codeAdditions)}</span>
+          <span class="text-red-700 select-none">-{fmt.Sprintf("%v", codeDeletions)}</span>
+        </p>        
+        if showTime {
+				<svg viewBox="0 0 2 2" class="h-0.5 w-0.5 fill-current">
+					<circle cx="1" cy="1" r="1"></circle>
+				</svg>
+				<p><time datetime={ time }>{ timeAgo }</time></p>
```

Comment:
@dejan-crocoder I would drop this time ago. I don't think is necessary.

Reply:
removd